client/my-sites/plans-features-main/components/pressable-promo-banner.tsx

```diff
@@ -17,10 +17,31 @@ const Banner = styled.div`
 	padding: 24px;
 	align-items: center;
 	gap: 32px;
+	max-width: 392px;
 
 	${ plansBreakSmall( css`
 		flex-direction: row;
 	` ) }
+
+	@media ( min-width: 440px ) {
```

Comment:
Not particularly proud of duping the width configuration of the plans grid here. A more consistent and reliable way would need a DOM restructure in my view, e.g. encapsulating the banner within the same container as the plans grid. However, there is only ~70+ clicks per day now, so I don't think we need to strive for that at the moment.